README.md

```diff
@@ -58,20 +22,14 @@ After you're finished please remove all the comments and instructions!
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 [MotoMate] <a name="about-project"></a>
 
-**[your_project__name]** is a...
+- Motomate is a booking app that allows users to easily book and reserve motorcycles online. With Motomate, users can log in and access a wide range of motorcycles to choose from. You can check the back-end of this project [here.](https://github.com/NatanimA/MotoMate-Backend-Api)
```

Comment:
_note: This was also an issue in the backend project._

- [ ]  On the "License" section of your readme, the link is broken because it points to a file `./LICENSE` that does not exist in your project.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪